pkg/sqlancer/sqlancer.go

```diff
@@ -329,10 +329,9 @@ func (p *SQLancer) progress() {
 		var transformers []transformer.Transformer
 		if approach == approachNoREC {
 			transformers = []transformer.Transformer{transformer.NoREC}
-		} else {
+		}
+		if approach == approachTLP {
 			transformers = []transformer.Transformer{
-				// approachTLP contains transformer.NoREC
-				transformer.NoREC,
```

Comment:
I prefer to change `nodesGroup := transformer.Transform(transformers, selectAst, 3)` to `nodesGroup := transformer.Transform(transformers, selectAst, 1)` in line 341

Reply:
More transforming time makes it easy to stress databases. So I prefer to remove `transformer.NoREC` from TLP.

@Hexilee What's your opinion?

Reply:
may we add another option like `mixed`?

Reply:
If we can resolve such a case on the PR description, I think it's ok. But we cannot process it now.